### DISCUSSION TITLE
Schema for Satellite

### DISCUSSION THREAD

**Discussion by dselva80** [eose-tools-org/eose-api#6] [General]
For now, should contain an orbit and instrument FOV.

**Comment by ptgrogan** [comment]
I added a placeholder [satellite definition](https://github.com/eose-tools-org/eose-api/blob/dev-tatc-analysis/src/eose/satellites.py) along with the coverage analysis discussed in the other discussion topics.

**Comment by vinay0000** [comment]
In OrbitPy, we have the satellite as an entity defined with the following properties: 

[(1) Orbit state,](https://github.com/eose-tools-org/eose-api/discussions/4) 

[(2) Spacecraft bus,](https://github.com/eose-tools-org/eose-api/discussions/10) 

[(3) Instrument(s). ](https://github.com/eose-tools-org/eose-api/discussions/9)
